assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx

```diff
@@ -52,6 +52,7 @@ const InstructionView = ({
       bodyMaxLength={NO_BODY_LENGTH}
       draftable
       draftSuccessMsgId="debate.brightMirror.draftSuccessMsg"
+      fullscreen
```

Comment:
How would fullscreen be set upstream?

Reply:
@ayyazdaniaryan Doesn't need to go higher, it s for the TopFormContainer that is already a higher up component